plugins/de/opendiabetes/vault/plugin/importer/CSVImporter.java

```diff
@@ -116,32 +114,27 @@ public boolean processImport(final InputStream fileInputStream, final String fil
             }
             if (creader == null) { //header could not be validated
                 LOG.log(Level.WARNING, "No valid header found in File:{0}", filenameForLogging);
-                return false;
+                return null;
             }
             // read entries
             while (creader.readRecord()) {
                 /*here the method template is used to process all records */
                 List<VaultEntry> entryList = parseEntry(creader);
 
-                boolean entryIsInterpreted = false;
                 if (entryList != null && !entryList.isEmpty()) {
                     for (VaultEntry item : entryList) {
-                        item.setRawId(importedRawData.size()); // add array position as raw id
                         importedData.add(item);
```

Comment:
Hier könnte man anstatt des for-Loops auch einfach ein list.addAll() verwenden

Reply:
Aber hier ist ja auch noch ein log für jede zeile. Der würde damit auch rausfliegen.

Reply:
Joa stimmt, aber welcher Nutzen bringt dieses Log?

Reply:
Jo das weiß ich nicht, von mir aus kann der raus.

Reply:
Von mir aus auch